src/platforms/esp/8266/clockless_block_esp8266.h

```diff
@@ -11,7 +11,10 @@
 
 #define FIX_BITS(bits) (((bits & 0x0fL) << 12) | (bits & 0x30))
 
+#ifndef MIN
 #define MIN(X,Y) (((X)<(Y)) ? (X):(Y))
+#endif
```

Comment:
if we can be sure that fl/math_macros.h is always loaded before this file, we could just delete the MIN macro. but that would create a rather implicit dependency...

i am not too familiar with C best practices, so not sure what the right solution is. this certainly is less risky of BC breaks.